dotenv-linter/src/checks/quote_character.rs

```diff
@@ -31,6 +31,10 @@ impl Check for QuoteCharacterChecker<'_> {
             return None;
         }
 
```

Comment:
Please add a comment on which case we are handling here:
```suggestion
// Don't remove quotes for arrays of JSON strings, e.g.: '["a"]'
```

Reply:
I have added the comment! Thanks for the suggestion!